src/app/GameView.ts

```diff
@@ -84,10 +110,23 @@ export class GameView {
     return new BankBoard().renderBankBoard(bank);
   }
 
-  private setBackground(player: Player): void {
-    (document.querySelector(
-      '.page-container',
-    ) as HTMLElement).style.background = player.theColor;
+  private createAlertPanel(): HTMLElement {
+    let alertContainer = document.querySelector(
+      '.alert',
+    ) as HTMLElement;
+    if (!alertContainer) alertContainer = Alert.createElement();
+    // TODO: connect with other methods to display the right alert
+    Alert.updateAlert('Lorem ipsum dolor sei', AlertType.INFO);
+    return alertContainer;
+  }
+
+  private setColorAccents(player: Player): void {
+    document
+      .querySelectorAll('.player-panel__buttons .button')
+      .forEach((element) => {
+        (element as HTMLElement).style.borderColor = player.theColor;
+      });
+    // TODO: change to borders and font colors
```

Comment:
This is where the methods for changing colors come, so perhaps adding border to the current player board also